tsconfig.json

```diff
@@ -13,8 +13,8 @@
     /* Language and Environment */
     "target": "ES2019" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
     "lib": [
-      "ES2020",
-      "DOM"
+      "ES2019",
```

Comment:
With Deno we should be able to target ESNext

Reply:
Not just the `lib`, but target as well I imagine? And I can do the same in the demo site since it's deployed to our edge.

Reply:
Yeah. It would be good to check, but generally Deno is about as bleeding edge as it gets in their syntax support

Reply:
And I've done it again! This is compiling for Node, not Deno 🤦🏻‍♂️

Reply:
Been there haha